infrastructure/quick-deploy/aws/eks/variables.tf

```diff
@@ -75,6 +75,14 @@ variable "eks" {
       scale_down_delay_after_failure        = string
       scale_down_unneeded_time              = string
       skip_nodes_with_system_pods           = bool
+      version                               = optional(string, "9.24.0")
```

Comment:
Do we allow optional for this deployment? I think this deployment should be kept compatible with versions of terraform before 1.3